man/io_uring_setup.2

```diff
@@ -106,19 +106,29 @@ call with the following code sequence:
  * and the kernel might not agree on the consistency of the
  * wakeup flag.
  */
-unsigned flags = atomic_load_relaxed(sq_ring->flags);
+unsigned flags = atomic_load_relaxed(ring->sq.kflags);
 if (flags & IORING_SQ_NEED_WAKEUP)
     io_uring_enter(fd, 0, 0, IORING_ENTER_SQ_WAKEUP);
 .EE
 .in
 
 where
-.I sq_ring
+.I ring->sq
 is a submission queue ring setup using the
 .I struct io_sqring_offsets
 described below.
 .TP
 .BR
+Note that, when using liburing, you never directly call the
+.I io_uring_enter()
+system call. That is usually taken care of by liburing’s
+.I io_uring_submit()
+function. It automatically determines if you are using
+polling mode or not and deals with when your program needs to call
```

Comment:
polling is so overloaded. Could refer to a poll request, sqpoll, or iopoll. I would spell this out specifically, that you're talking about a ring setup with IORING_SETUP_SQPOLL here.